docs/how-tos/sensor-data-query-sdk.md

```diff
@@ -64,6 +64,16 @@ source .venv/bin/activate
 pip install viam-sdk
 ```
 
+{{% /tablestep %}}
+{{% tablestep%}}
+**2. Install requirements**
+
+To query data with the Python SDK, you will need `bson` from the `pymongo` package. To install `bson`, run the following command:
```

Comment:
This is incorrect. You either need the `bson` package or the `pymongo` package. The command you are running here installs the bson package.
```suggestion
To query data with the Python SDK, you will the `bson` package or the `pymongo` package.
To install `bson`, run the following command:
```

Reply:
sorry i didn't realize i was excluding the possibility of having the pymongo package! just wanted to point to simplest solution